samples/functions2.go

```diff
@@ -28,5 +28,5 @@ func location(city string) (lat, lng float64) {
 func main() {
 	fmt.Println("sum(2,5)", sum(2, 5))
 	fmt.Println(location("Bangalore"))
-	fmt.Println(gimmiTwo())
+	fmt.Println(gimmeTwo())
```

Comment:
speaker notes: issues with nesting function calls where the inner returns multiple values. you can't pass any other values.